README.md

```diff
@@ -119,6 +119,37 @@ You will have a file called flipper.contract. You can use this directly in
 the [Polkadot UI](https://substrate.dev/substrate-contracts-workshop/#/0/deploy-contract),
 as if your smart contract was written using ink!.
 
+## Roadmap
+
+Solang has a long way to go in order to achieve full compatibility for the blockchains we target.
```

Comment:
I don't think we should downplay what solang can already do. It's really not *that* far from full compatibility.